gateway/platforms/vertx3/vertx3/src/main/java/io/apiman/gateway/platforms/vertx3/http/HttpExecutor.java

```diff
@@ -166,7 +164,9 @@ private void setError(Throwable error) {
         response.setStatusCode(HttpResponseStatus.INTERNAL_SERVER_ERROR.code());
         response.setStatusMessage(HttpResponseStatus.INTERNAL_SERVER_ERROR.reasonPhrase());
         response.headers().add("X-Exception", String.valueOf(error.getMessage())); //$NON-NLS-1$
-        response.write(ExceptionUtils.getStackTrace(error));
+        response.headers().add(HttpHeaders.CONTENT_TYPE,  MediaType.APPLICATION_JSON);
+        response.write(String.format("{\"responseCode\":%d, \"message\":\"%s\"}",
+                HttpResponseStatus.INTERNAL_SERVER_ERROR.code(), error.getMessage()));
```

Comment:
I was printing the stack trace before, but it was failing the tests because of it. I swear I'd done this before but somehow I must have lost it/reverted it. 

Anyway, will fix.